website/merchandise/templates/merchandise/product_page.html

```diff
@@ -0,0 +1,51 @@
+{% extends 'simple_page.html' %}
+{% load static i18n thumbnail %}
+
+{% block title %}{% trans "Product" %} — {{ block.super }}{% endblock %}
+{% block opengraph_title %}{% trans "Merchandise" %} — {{ block.super }}{% endblock %}
+
+{% block page_title %}{{product.name}}{% endblock %}
+{% block page_content %}
+{% block css_head %}
+    {{ block.super }}
+    <link href="{% static 'merchandise/css/product.css' %}" rel="stylesheet" type="text/css">
+{% endblock %}
+
+
+<!DOCTYPE html>
```

Comment:
You don't need these headers as theyre already there by inheritting from simple_page.html